modules/ROOT/pages/dataweave.adoc

```diff
@@ -6,7 +6,7 @@ endif::[]
 
 DataWeave is the MuleSoft expression language for accessing and transforming data received through a Mule app. DataWeave is tightly integrated with Mule runtime, which runs the scripts and expressions in your Mule app.
 
-DataWeave acts on data in the Mule event. Most commonly, you use it to access and transform data in the message payload. For example, after a component in your app retrieves data from one system, you can use DataWeave to modify and output selected fields in that data to a new data format, then use another component in your app to pass on that data to another system.
+DataWeave acts on data in the Mule event. Most commonly, you use it to access and transform data in the Mule message payload. For example, after a component in your app retrieves data from one system, you can use DataWeave to modify and output selected fields in that data to a new data format, then use another component in your app to pass on that data to another system.
```

Comment:
Why add Mule here?

Reply:
To emphasize that the payload is part of the Mule message object.

Reply:
How could it be anything but a Mule message payload in a Mule event?